docs/API.md

```diff
@@ -45,6 +45,10 @@ Write a packet to all `clients` but encode it only once.
 
 Verifies if player's chat message packet was signed with their Mojang provided key
 
+### client.logSentMessageFromPeer(packet)
```

Comment:
Why isn't this done internally by the nmp client?

Reply:
This one is for a client connected to a server, so the way to handle this internally would be to capture `client.write('player_chat')` as it gets sent to the server and do the handling there. That seemed too hacky so I added a separate call that server code needs to explicitly call into before broadcasting a player message to other players

Reply:
An alternative could be to replace this with a "sendMessageFromPeer" function taking in a chat_message received from another player, but the payload between chat_message (serverbound) and player_chat (clientbound) is pretty different. Since chat formatting / filters and some other extra fields exist in the player_chat packet. So I thought it made sense to not overcomplicate nmp with lots of handling for this and just let the library users handle that and just call into this just for verification bookkeeping after sending off a message

Reply:
Ok makes sense, let's keep it that way